### PR TITLE
Guardian - Offensive Ironfur

### PR DESCRIPTION
**Pull request by syrifgit**
Also fixed ironfur duration formula

### CODE REVIEW

File: TheWarWithin/DruidGuardian.lua [hunk 1]
```diff
@@ -488,7 +488,7 @@ spec:RegisterAuras( {
     -- https://wowhead.com/beta/spell=192081
     ironfur = {
         id = 192081,
-        duration = function() return ( buff.guardian_of_elune.up and 9 or 7 ) + talent.ursocs_endurance.rank end,
+        duration = function() return 7 + ( buff.guardian_of_elune.up and 3 or 0 ) + ( talent.ursocs_endurance.rank and 2 ) end,
```
Comment (Hekili) [inline TheWarWithin/DruidGuardian.lua]: Needs to be `+ ( talent.ursocs_endurance.rank * 2 )` or `+ ( talent.ursocs_endurance.enabled and 2 or 0 )`

Comment (syrifgit) [reply TheWarWithin/DruidGuardian.lua]: The talent doesn't have ranks anymore.
image

Comment (Hekili) [reply TheWarWithin/DruidGuardian.lua]: All talents have ranks.  Some have multiple ranks.  If Ursoc's Endurance is enabled, `talent.ursocs_endurance.rank` equals `1`, `1 * 2` = `2` seconds.  If it is not enabled, `talent.ursocs_endurance.rank` equals `0`, `0 * 2` = `0` seconds.

Comment (syrifgit) [reply TheWarWithin/DruidGuardian.lua]: nvm I understand what youre saying now

**Review by Hekili — APPROVED** [review]

**Pull request merged**